src/components/App.tsx

```diff
@@ -999,9 +1016,7 @@ function App() {
                 borderBottomWidth="1px"
               >
                 <NavigationBar
-                  newUserNodeLinkedToANewSystemNode={() =>
-                    newUserNodeLinkedToANewSystemNode()
```

Comment:
i think we had good reason for doing this. `newUserNodeLinkedToANewSystemNode` takes an argument and onClick would pass an event to it that screws things up